dbms/src/Common/RandomData.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <common/types.h>
+
+String randomString(UInt64 length);
```

Comment:
Put these utils function in the root namespace?

Reply:
What does root namespace mean? I think it is in the root namespace now.

Reply:
It is better to move these functions to TestUtils and adding `DB::Test` namespace to avoid conflict definition with thirdparth libs

Reply:
I would like use it to generate random key, so not just for tests

Reply:
Adding a namespace for these functions is better, for example `DB::random`

Reply:
done